lib/matplotlib/tests/test_pyplot.py

```diff
@@ -371,23 +370,13 @@ def test_set_current_axes_on_subfigure():
 
 
 def test_pylab_integration():
-    pytest.importorskip("IPython")
-    subprocess.run(
-        [
-            sys.executable,
-            "-m",
-            "IPython",
-            "--pylab",
-            "-c",
-            ";".join((
-                "import matplotlib.pyplot as plt",
-                "assert plt._REPL_DISPLAYHOOK == plt._ReplDisplayHook.IPYTHON",
-            )),
-        ],
-        env={**os.environ, "SOURCE_DATE_EPOCH": "0"},
+    mpl.testing.subprocess_run_helper(
+        pytest.importorskip("IPython").start_ipython,
```

Comment:
```suggestion
    IPython = pytest.importorskip("IPython")
    mpl.testing.subprocess_run_helper(
        IPython.start_ipython,
```
Slight preference for separtion of import and usage. While it works as is, the skipping is more clear when using isolated at the top of the file instead of inlined in a complex expression.

Reply:
sure